concourse/scripts/kibana_set_utc.rb

```diff
@@ -0,0 +1,76 @@
+#! /usr/bin/env ruby
+
+require 'net/http'
+require 'json'
+
+def config_uri(es_url)
+  config_url = "#{es_url}/.kibana/config/4.3.1"
+  URI(config_url)
+end
+
+def index_settings_uri(es_url)
+  index_settings_url = "#{es_url}/.kibana"
+  URI(index_settings_url)
+end
+
+def update_es_index(uri, document)
+  send_request(
+    uri: uri,
+    http_method: :Put,
+    body: document,
+    allowed_response_codes: ['200', '201']
+  )
+end
+
+def send_request(uri:, http_method:, allowed_response_codes:, body: nil)
+  raise "Document is not a hash: #{body}" unless body == nil || body.is_a?(Hash)
+  response = nil
+  Net::HTTP.new(uri.host, uri.port).start do |http|
+    method = Net::HTTP.const_get(http_method)
+    headers = body ? { 'Content-Type' => 'application/json' } : {}
```

Comment:
The intention here is to set the headers - we only send a body in json format, so send the appropriate header if the body was given, or send no headers if it wasn't